analyzers/src/SonarAnalyzer.Common/Rules/CertificateValidationCheckBase.cs

```diff
@@ -308,7 +309,9 @@ private static ImmutableArray<TInvocationExpressionSyntax> FindInvocationList(Sy
             var ret = ImmutableArray.CreateBuilder<TInvocationExpressionSyntax>();
             foreach (var invocation in root.DescendantNodesAndSelf().OfType<TInvocationExpressionSyntax>())
             {
-                if (c.SemanticModel.GetSymbolInfo(invocation).Symbol.Equals(method))
+                if (AreSameMethods(invocation, method.Name)
+                    && c.SemanticModel.GetSymbolInfo(invocation).Symbol is { } symbol
```

Comment:
With the name check in place, `null` should be recoverable with an undefined method and some `// Error` flag.